tests/test_conditions.py

```diff
@@ -37,6 +44,19 @@ def make_mock_player(
     )
 
 
+def make_mock_vip_player(steam_id_64: str, expiration_date: datetime | None = None):
+    return VipPlayer(
+        player=make_mock_player(steam_id_64=steam_id_64),
+        expiration_date=expiration_date,
+    )
+
+
+def make_moc_get_vips_dict(vips: list[VipPlayer]) -> dict[str, VipPlayer]:
```

Comment:
`mock` not `moc`

Reply:
Oops. Also updated and used the function...